core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -144,8 +164,24 @@ private TabletsMetadata buildNonRoot(AccumuloClient client) {
       }
     }
 
+    private void configureColumns(ScannerBase scanner) {
+      for (Text fam : families) {
+        scanner.fetchColumnFamily(fam);
+      }
+
+      for (ColumnFQ col : qualifiers) {
+        col.fetch(scanner);
+      }
+
+      if (families.isEmpty() && qualifiers.isEmpty()) {
+        fetchedCols = EnumSet.allOf(ColumnType.class);
+      }
+    }
+
     @Override
     public Options checkConsistency() {
+      Preconditions.checkState(extents == null,
```

Comment:
I like to statically import Preconditions.checkState (or Preconditions.checkArgument). It can help with readability.

Reply:
f017626